### PR TITLE
DCS-1187 displays prisoner's actual image or a placeholder image

### PR DESCRIPTION
**Pull request by GurnankCheema**
A prisoner's image, or a placeholder if image not available, will be displayed. 

Screenshot 2021-09-24 at 11 13 44

### CODE REVIEW

**Review by andrewrlee — COMMENTED** [review]
LGTM but needs some level of testing

Could add cypress test to return an image from wiremock and potentially verify it's loaded when accessing content

File: server/routes/index.ts [hunk 1]
```diff
@@ -19,5 +20,8 @@ export default function routes(services: Services): Router {
   const temporaryAbsencesController = new TemporaryAbsencesController(services.temporaryAbsencesService)
   get('/confirm-arrival/return-from-temporary-absence', temporaryAbsencesController.view())
 
+  const imageController = new ImageController(services.incomingMovementsService)
```
Comment (andrewrlee) [inline server/routes/index.ts:23]: could maybe call it prisoner or offender controller? 

Image is quite generic to be sitting on such a specific url

Comment (GurnankCheema) [reply server/routes/index.ts:23]: renamed to prisonerController

**Review by connormaglynn — APPROVED** [review]
LGTM 😃

**Pull request merged**